p2p/client.go

```diff
@@ -165,6 +203,65 @@ func (c *Client) GossipBlock(ctx context.Context, blockBytes []byte) error {
 	return c.blockGossiper.Publish(ctx, blockBytes)
 }
 
+// AddBlock stores the block in the block-sync datastore, stores locally the returned identifier and advertises the identifier to the DHT, so other nodes can know the identifier for the block height.
+func (c *Client) AddBlock(ctx context.Context, height uint64, blockBytes []byte) error {
```

Comment:
I'd suggest `SaveBlock` to be aligned with current store convention:
`Save` -> saving a data piece to store
`Load` -> loading a data piece from store
`Delete` -> deleting a data piece from store

Reply:
funcs renamed